VerifyBamID/2.0.1--h32f71e1_0/CheckContamination.nf

```diff
@@ -0,0 +1,64 @@
+process CheckContamination {
+    tag {"VerifyBamID CheckContamination ${sample_id}.${int_tag}"}
+    label 'VERIFYBAMID_2_0_1_h32f71e1_0'
+    label 'VERIFYBAMID_2_0_1_h32f71e1_0_CheckContamination'
```

Comment:
In this case, I did it on purpose because we use Verifybamid + part of python code. Therefore renaming the process to match with tool name, might be misleading. 
But this is influenced by the discussion about keeping the python code and where it belongs. Tbc.